pkg/tfbridge/config_encoding.go

```diff
@@ -164,12 +134,22 @@ func (enc *configEncoding) UnmarshalPropertyValue(key resource.PropertyKey, v *s
 		return &v, nil
 	}
 
+	// Computed sentinels are coming in as always having an empty string, but the encoding coerses them to a zero
```

Comment:
This might someday help avoid typing mistakes in code that parses PropertyValue but for now it's not terribly important; if something's a Computed it's not looked at what's inside.